### ISSUE TITLE
Remove editable decoration when bluring inline

### ISSUE THREAD

**Issue by mlewand** [cksource/ckeditor-plugin-a11ychecker#38]
We should remove all the a11ychecker markup from editable when user blurs the inline editor.
1. Open any sample with Accessibility Checker in inline editor. (i.e. [plugins/a11ychecker/dev/sample.html](http://ckeditor.dev/plugins/a11ychecker/dev/sample.html))
2. Enable Accessibility Checker.
3. When balloon will appear, simply click outside of the editable - so editor is blurred.

**Expected result:**
All the extra markup produced by Accessibility Checker (with an exception of `data-a11y-ignore` attr) should be removed.

**Current result:**
All the classes added by checker are still there, so eg. issues are still highlighted.

Additional info:
1. But, we need to be careful here I can see that, eg. after applying a QuickFix in inline editor, whole toolbar flickers. That tells me that blur occurs there.
